### ISSUE TITLE
Support other integer scales > 2

### ISSUE THREAD

**Issue by Kekun** [cassidyjames/dippi#128]
Let's say I have a device where a scale of 300% would be ideal, currently Dippi tells it's "Too High DPI", when it would be perfect for High DPI, I just need to use it *@3x*. That would be nice if Dippi supported integer scales greater than 2.

**Comment by cassidyjames** [comment]
Yep, agreed, that would make sense. Probably in scope for a redesign (#122), but could maybe be bolted on as-is.
